### PR TITLE
[TT-1190] support multiple configuration names

### PR DESCRIPTION
**Pull request by Tofel**
Adds the following change:
```
config, err = tc.GetConfig([]string{"A", "B", "C"}, tc.VRFv2Plus)
require.NoError(t, err, "Error getting config")
```

which means that in this case the overriding order would be:
* configuration without any name
* "A" configuration
* "B" configuration 
* "C" configuration

If config had this structure:
```
struct Config type {
     Text_1 string `toml:"text_1"`
     Text_2 string `toml:"text_2"`
     Text_3 string `toml:"text_3"`
     Text_4 string `toml:"text_4"`
}
```

and if TOML looked like this:
```
text_1 = "first"
text_2 = "second"
text_3 = "third"
text_4 = "original"

[A]
text_1 = "overwritten_from_A"

[B]
text_2 = "overwritten_from_B"

[C]
text_3 = "overwritten_from_C"
```

The result would be:
```
text_1 = "overwritten_from_A"
text_2 = "overwritten_from_B*"
text_3 = "overwritten_from_C"
text_4 = "original"
```

### CODE REVIEW

**Comment by Tofel** [comment]
needs OK from @iljapavlovs before merging

**Review by gheorghestrimtu — DISMISSED** [review]

**Comment by gheorghestrimtu** [comment]
`[]string` ftw

**Review by skudasov — DISMISSED** [review]

**Review by iljapavlovs — APPROVED** [review]

**Review by gheorghestrimtu — APPROVED** [review]

**Review by anirudhwarrier — APPROVED** [review]

**Pull request merged**
